src/BlockEntities/BrewingstandEntity.h

```diff
@@ -119,6 +126,9 @@ class cBrewingstandEntity :
 	/** Amount of ticks that the current item has been brewed */
 	short m_TimeBrewed;
 
+	/** The fuel for the brewing stand */
+	short m_Fuel;
```

Comment:
What's `the fuel`, is it the item being consumed, or some entity? How about renaming to `FuelTicksLeft` (and all the related functions renamed accordingly)?